demo/docbot/cluster_bootstrap.py

```diff
@@ -51,7 +51,7 @@ def initialize_cluster_settings(client: OpenSearch) -> None:
         raise Exception("Failed to initialize cluster settings.")
 
 
-def init_index_template(client: MLClient, template_name = "nlp-template") -> None:
+def init_index_template(client: MLClient, template_name = "nlp-template", template_data=None) -> None:
```

Comment:
Here we can drop template data as I think template name may be the only variable. We may pull out index patterns as well. I'll leave that decision to you.